linguee/__init__.py

```diff
@@ -0,0 +1,105 @@
+# -*- coding: utf-8 -*-
+
+"""linguee extension
+
+translate ger-eng with linguee
+
+Synopsis: <trigger> <word>"""
+
+
+from albertv0 import *
+import requests
+import bs4
+import os
+
+
+lang = "deutsch-englisch"
```

Comment:
is there a way to let the ext recognize this on its own? if upstream then it should be internationlaized.

Reply:
It would be possible to make the user specify a language like "lin de some word" or "lin it qualche parola". I would however like to keep english as the default second language because linguee might not support translations from any arbitrary language into another and also the prompt would become more cluttered (imagine "lin de it durcheinander").
Would this be okay?